src/github.com/getlantern/lantern-ui/app/partials/footer.html

```diff
@@ -25,6 +25,12 @@
     </div>
   </div>
 
+  <div id="share">
+      <div id="twitter"><a class="fa fa-twitter share-base share-twitter" onclick="window.open('https://twitter.com/intent/tweet?text=I\'m using Lantern -- an app for defeating Internet censorship http://getlantern.org via @getlantern')"><img width="30px" height="30px" src="/img/twitter_icon.png"></a></div>
+      <div id="fb"><a class="fa fa-facebook share-base share-facebook" onclick="window.open('https://www.facebook.com/dialog/share?app_id=1562164690714282&display=popup&href=https://getlantern.org&redirect_uri=https://getlantern.org')"><img width="30px" height="30px" src="/img/fb_icon.png"></a></div>
```

Comment:
I'm getting an error clicking this one @atavism -- on FB: "App Not Setup: This app is still in development mode, and you don't have access to it. Switch to a registered test user or ask an app admin for permissions."